Hudl.Mjolnir.Tests/Command/Attribute/NewCommandAttributeAndProxyTestsIgnoringTimeouts.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Threading;
+using Hudl.Mjolnir.Command.Attribute;
+using Hudl.Mjolnir.Tests.Helper;
+using Xunit;
+
+namespace Hudl.Mjolnir.Tests.Command.Attribute
+{
+    public sealed class NewCommandAttributeAndProxyTestsIgnoringTimeouts : TestFixtureIgnoreTimeouts
```

Comment:
> In each test, since we don't know what other tests may have done to the ConfigProvider

If we assume that other tests appropriately tore themselves down, is that enough?

> we still need to start the test off with a ...

If we can assume a sane config state, and the config value defaults to `false`, we should only have to add it to cases where we want to set it with a `true` value.

Thoughts there?